### PR TITLE
Development

### PR DESCRIPTION
**Pull request by RadiactiveJesus**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  resources :users, only:[:new, :create, :edit]
```
Comment (azdrenymeri) [inline config/routes.rb]: You also need an `:update`  route

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  def create
+    @user = User.new(user_params)
+    
+    if @user.save
+      redirect_to new_user_path
+    else
+      render :new
+    end
+  end
+  def user_params
```
Comment (azdrenymeri) [inline app/controllers/users_controller.rb]: the `user_params` method should be added at the end of the class and also should be set as `private`

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new()
```
Comment (azdrenymeri) [inline app/controllers/users_controller.rb]: small braces are unnecessary without arguments

File: app/controllers/users_controller.rb [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  end
+
```
Comment (azdrenymeri) [inline app/controllers/users_controller.rb]: Where is your `update` action?

File: app/views/users/new.html.erb [hunk 5]
```diff
@@ -0,0 +1,27 @@
+<!--<form accept-charset="UTF-8" action="/users" method="POST">
```
Comment (azdrenymeri) [inline app/views/users/new.html.erb]: on the `action` attribute of the form, you should use the rails URL helpers not a static one

File: app/views/users/new.html.erb [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+    <label id = "username" for ="username">username</label>
+    <input type="text" name="user[username]"><br>
+    <label id = "email" for ="email">email</label>
+    <input type="text" name="user[email]"><br>
+    <label id = "password" for ="password">pasword</label>
+    <input type="password" name="user[password]"><br>
+    <input type="submit" value="Submit">      
+</form>-->
+
+<!--<%= form_tag("/users", method: "post") do %>
```
Comment (azdrenymeri) [inline app/views/users/new.html.erb]: Please refer line 1 of this file for setting your URL to your forms

Comment (RadiactiveJesus) [reply app/views/users/new.html.erb]: We at the momento of making the forms work figured out that this is the way the forms are working for us, we fixed the rest of the issues, can you explain more in depth how to and how is it useful doing it this way? thanks

**Pull request merged**